### DISCUSSION TITLE
Detekt 1.23.4 Rule Set PDF

### DISCUSSION THREAD

**Discussion by YektaDev** [detekt/detekt#6759] [Show and tell]
Hey folks,

I've converted the complete rule set of Detekt 1.23.4 into a PDF _(to partially present at the Software Engineering class in uni)_, excluding the "Configuration options" sections. Maybe someone here could find it handy as a quick reference or just useful in general 🙂.

Here's the [link](https://github.com/detekt/detekt/files/13768317/Detekt.pdf) to the PDF.

Feel free to grab it if it might be of use to you!

Cheers

**Comment by schalkms** [comment]
Cool! Glad to hear that detekt gains traction in academia as well. 👍

**Comment by cortinico** [comment]
This is awesome <3
